### PR TITLE
Fix container error

### LINKED ISSUE #501: Need to revisit: ContainerError > Either Tried resolving the too many dependencies
That error is thrown a lot because of an if statement in the resolver. This needs to be fixed and either throw one or the other

### PR DESCRIPTION
**Pull request by josephmancuso**
This completely removed the beginning text which seemed to be always incorrect.

I think the "tried resolving too many dependencies" thing was something i encountered mid development. I can't seem to replicate that use case. I remember coding and then refactoring a bit so maybe that is no longer the case anymore.

Closes #501

### CODE REVIEW

File: masonite/app.py [hunk 1]
```diff
@@ -223,6 +223,9 @@ def _find_annotated_parameter(self, parameter):
         raise ContainerError(
             'The dependency with the {0} annotation could not be resolved by the container'.format(parameter))
 
+    def get_parameters(self, obj):
```
Comment (vaibhavmule) [inline masonite/app.py]: Where this will get used?

Comment (josephmancuso) [reply masonite/app.py]: just updated

Comment (josephmancuso) [reply masonite/app.py]: :)

Comment (vaibhavmule) [reply masonite/app.py]: oh lol, I jumped in the middle of your work.

Comment (josephmancuso) [reply masonite/app.py]: np lol. I was actually done but we caught that at the exact same time and I updated PR

**Pull request merged**